### PR TITLE
[FS-172266] : Fixing conductor prom metrics

### PR DESCRIPTION
**Pull request by keerthivaasan-kanagaraj**
Pull Request type
----
- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes (Please run `./gradlew generateLock saveLock` to refresh dependencies)
- [ ] WHOSUSING.md
- [ ] Other (please describe):

**NOTE**: Please remember to run `./gradlew spotlessApply` to fix any format violations.

Changes in this PR
----

_Describe the new behavior from this PR, and why it's needed_
Issue #

Alternatives considered
----

_Describe alternative implementation you have considered_

### CODE REVIEW

File: server/src/main/java/com/netflix/conductor/config/PrometheusIntegrationConfig.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import io.micrometer.core.instrument.Metrics;
+
+// This class loads all the configurations related to prometheus. 
+@Configuration
+public class PrometheusIntegrationConfig
+        implements CommandLineRunner {
+
+    private static final Logger log = LoggerFactory.getLogger(PrometheusIntegrationConfig.class);
+
+    @Autowired
```
Comment (ab48917) [inline server/src/main/java/com/netflix/conductor/config/PrometheusIntegrationConfig.java]: @Autowired is deprecated. Please move to Constructor based wiring

File: core/src/main/java/com/netflix/conductor/metrics/WorkflowMonitor.java [hunk 2]
```diff
@@ -77,48 +77,57 @@ public WorkflowMonitor(
...
-            taskDefs.forEach(
-                    taskDef -> {
+        }
+
+        // This getPendingWorkflowCount is not implemented in scylla execution DAO class. 
+        // Disabling this temporarily.
+        // try {
+        //         getPendingWorkflowToOwnerAppMap(workflowDefs)
+        //                 .forEach(
+        //                         (workflowName, ownerApp) -> {
```
Comment (ab48917) [inline core/src/main/java/com/netflix/conductor/metrics/WorkflowMonitor.java:91]: have added this in staging_debug PR along with its refresh config. Please ignore this,

File: core/src/main/java/com/netflix/conductor/metrics/Monitors.java [hunk 3]
```diff
@@ -392,8 +392,11 @@ public static void recordWorkflowCompletion(
     }
 
     public static void recordUnackTime(String workflowType, long duration) {
```
Comment (ab48917) [inline core/src/main/java/com/netflix/conductor/metrics/Monitors.java]: Can we. disable the entire method using a ConditionalOnBean ?

File: core/src/main/java/com/netflix/conductor/metrics/WorkflowMonitor.java (hunk 2 above)
Comment (keerthivaasan-kanagaraj) [reply core/src/main/java/com/netflix/conductor/metrics/WorkflowMonitor.java:91]: done

**Review by ab48917 — APPROVED** [review]

**Pull request merged**